core/templates/dev/head/pages/story-editor-page/editor-tab/story-node-editor.directive.ts

```diff
@@ -131,9 +136,15 @@ angular.module('oppia').directive('storyNodeEditor', [
           };
 
           $scope.updateExplorationId = function(explorationId) {
-            StoryUpdateService.setStoryNodeExplorationId(
-              $scope.story, $scope.getId(), explorationId);
-            $scope.currentExplorationId = explorationId;
+            ExplorationIdValidationService.isExpIdValid(
```

Comment:
Can you explicitly add a check for empty exploration ID here, and throw an error or alert? Since saving an empty field shows an orange error message from the backend.

Reply:
Changed the regex.